DESCRIPTION

```diff
@@ -24,7 +24,7 @@ BugReports: https://github.com/GSK-Biostatistics/tfrmt/issues
 Depends: 
     R (>= 4.1.0)
 Imports: 
-    cards (>= 0.6.1.9010),
+    cards,
```

Comment:
this still needs a minimum version because shuffle_card doesn't work with cards 0.4.0

Reply:
<img width="647" height="144" alt="image" src="https://github.com/user-attachments/assets/906e25ad-90d3-4788-b0c1-4e65a5a11743" />